pkg/api/job_runs.go

```diff
@@ -211,6 +211,17 @@ func JobRunRiskAnalysis(dbc *db.DB, jobRun *models.ProwJobRun, jobRunTestCount i
 				}
 			}
 
+			// otherwise, what is our best match...
+			// do something more expensive and check to see
+			// which tr contains all the variants we have currently
+			for _, tr := range trs {
```

Comment:
trs = test results? Please say test result at least in the comment, or [change the variable names](https://blogg.itverket.no/dont-think-shortening-variable-names-2/)

Reply:
sure, just copied what was above but I can refactor both